src/test/java/com/ontotext/kafka/mocks/DummyErrorHandler.java

```diff
@@ -0,0 +1,15 @@
+package com.ontotext.kafka.mocks;
+
+import com.ontotext.kafka.error.ErrorHandler;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+public class DummyErrorHandler implements ErrorHandler {
+	@Override
+	public void handleFailingRecord(SinkRecord record, Throwable ex) {
```

Comment:
You can introduce some sort of collection to store the records handled and verify them as well as the throwables associated with them.

This will help you cover even more logic with your unit tests.